jgrapht-core/src/test/java/org/jgrapht/alg/treedecompostion/NiceDecompositionUtil.java

```diff
@@ -0,0 +1,114 @@
+package org.jgrapht.alg.treedecompostion;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.*;
+import java.util.Map.*;
+
+import org.jgrapht.*;
+import org.jgrapht.graph.*;
+
+public class NiceDecompositionUtil
```

Comment:
it is better to make utility class static, then you don't need to declare private constructor any more.